docs/source/getting_started/changelog.rst

```diff
@@ -10,6 +10,16 @@ See the `JupyterLab
 3.0 <https://github.com/jupyterlab/jupyterlab/milestone/48?closed=1>`__
 milestone on GitHub for the full list of pull requests and issues closed.
 
+v3.0.13
+^^^^^^^
+* Add cell id per notebook format 4.5 (`#10018 <https://api.github.com/repos/jupyterlab/jupyterlab/pulls/10018>`__)
```

Comment:
This and some of the other links are to API.github.com, which just dumps raw json into the browser.